README.md

```diff
@@ -1,87 +1,100 @@
+![](docs/images/logo.png "Project Atomic")
 
-Examples of this tool may be found within the [Nulecule library repo](https://github.com/projectatomic/nulecule/tree/master/examples).
+Atomic App is a reference implementation of the [Nulecule](https://github.com/projectatomic/nulecule) specification. Packaged Atomic App containers are "Nuleculized" and each component of the package is a "Nulecule".
 
-## Getting Started
+Atomic App is used to bootstrap packaged container environments and run them on multiple container orchestrators. It is designed from the ground-up to be portable and provider pluggable.
 
-Atomic App itself is packaged as a container. End-users typically do not install the software from source. Instead use the `atomicapp` container as the `FROM` line in a Dockerfile and package your application on top. For example:
+  - __A "packaged installer" for all container-based environments and applications.__ Replace all those bash and Ansible scripts with one container-based deployment tool.
 
-```
-FROM projectatomic/atomicapp
-
-MAINTAINER Your Name <you@example.com>
-
-ADD /Nulecule /Dockerfile README.md /application-entity/
-ADD /artifacts /application-entity/artifacts
-```
+  - __Target multiple providers:__ Specify the provider you want the Atomic App to run on. It supports Kubernetes, OpenShift, Mesos+Marathon and Docker.
 
-For more information see the [Nulecule getting started guide](https://github.com/projectatomic/nulecule/blob/master/docs/getting-started.md).
+  - __Inherit already packaged containers:__ Create composite applications by referencing other Nulecule-compliant applications. For example, plugging in an alternative well-orchestrated database in another referenced container image.
 
-## Developers
+  - __Fetch and run entire environments:__ Use `atomicapp fetch` and `atomicapp run` to run pre-packaged Nuleculized containers.
 
-First of all, clone the github repository: `git clone https://github.com/projectatomic/atomicapp`.
-
-### Installing Atomic App locally
-Simply run
-
-```
+## Installing Atomic App
+From Linux:
+```sh
+git clone https://github.com/projectatomic/atomicapp && cd atomicapp
 make install
 ```
 
-If you want to do some changes to the code, I suggest to do:
+_or_ 
 
-```
-cd atomicapp
-export PYTHONPATH=`pwd`:$PYTHONPATH
-alias atomicapp="python `pwd`/atomicapp/cli/main.py"
+Download a pre-signed .tar.gz from [download.projectatomic.io](https://download.projectatomic.io) / [GitHub](https://github.com/projectatomic/atomicapp/releases):
+```sh
+export RELEASE=0.4.2
+wget https://github.com/projectatomic/atomicapp/releases/download/$RELEASE/atomicapp-$RELEASE.tar.gz
+tar -xvf atomicapp-$RELEASE.tar.gz && cd atomicapp-$RELEASE.tar.gz
+make install
 ```
 
-### Building for containerized execution
-```
-docker build -t [TAG] .
-```
+## Documentation
 
```

Comment:
Maybe say this:

This README contains some high level overview information on Atomic App. The detailed documentation for Atomic App resides in the docs directory. Links to each section have been conveniently listed below: